sklearn/metrics/cluster/_supervised.py

```diff
@@ -372,6 +373,12 @@ def adjusted_rand_score(labels_true, labels_pred):
       >>> adjusted_rand_score([0, 0, 0, 0], [0, 1, 2, 3])
       0.0
 
+    ARI may take a negative value for especially discordant labelings that
+    underperform compared to the expected value of random labels::
```

Comment:
Isn't "performant" rather used as a term to describe a method or an algorithm?

Is the following phrasing is a more exact?
```suggestion
    ARI may take a negative value for especially discordant labelings that
    are a worse choice than the expected value of random labels::
```